src/state/fetch-movies/fetch-movies.test.js

```diff
@@ -0,0 +1,29 @@
+import { fetchMovies } from "./fetch-movies";
+import { MovieService } from "../../services/movie-service";
+
+const mockMovieData = {
+  data: [{ id: 10 }],
+  totalAmount: 1,
+}
+
+describe('Fetch Movies', () => {
+  it('Calls dispatch 2 times', async () => {
```

Comment:
What's the difference between this test and the second one?

Reply:
Edited the tests to spy on action creators rather than mock dispatch. I think it would be clearer that way. Also changed the descriptions.